### PR TITLE
watchdog: add --pid-file

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: pkg/operator/watchdog/cmd.go [hunk 1]
```diff
@@ -164,8 +170,8 @@ func (o *FileWatcherOptions) Complete() error {
 }
 
 func (o *FileWatcherOptions) Validate() error {
-	if len(o.ProcessName) == 0 {
-		return fmt.Errorf("process name must be specified")
+	if len(o.ProcessName) == 0 && len(o.PidFile) == 0 {
+		return fmt.Errorf("process name or pid file must be specified")
```
Comment (mfojtik) [inline pkg/operator/watchdog/cmd.go]: nit: process name and pid file

Comment (sttts) [reply pkg/operator/watchdog/cmd.go]: no, "or" is correct.

**Pull request merged**